### ISSUE TITLE
904 or 940 nm

### ISSUE THREAD

**Issue by frillweeman** [hevnsnt/NOTCHACOTCHA#6]
In the defcon slides, you put both 904 and 940 nm for the wavelength of the light. One lidar vendor's specs say 904, and [Wikipedia](https://en.wikipedia.org/wiki/LIDAR_traffic_enforcement) says 905. Amazon link says 940, but won't that get rejected by the filter on the lidar gun (899-909 nm)?

**Comment by JeremyWhittaker** [comment]
Thanks for pointing this out. I already bought some of the 940. 
![Screenshot_20191225-120130](https://user-images.githubusercontent.com/13750325/71449653-8e617b00-270e-11ea-82eb-5f741baa1ee3.png)

**Comment by YipYup** [comment]
Correct. LIDAR operates at 904/905nm. 940nm is not the correct wavelength. Check [RDForum](https://www.rdforum.org) for more information.

**Comment by interbiznw** [comment]
So what your saying here is that what was built, presented and posted here will not work?

**Comment by interbiznw** [comment]
https://www.youtube.com/watch?v=QYpTsQLnW9o
